### DISCUSSION TITLE
ValueError: The following model_kwargs are not used by the model

### DISCUSSION THREAD

**Discussion by ZhenbinChan** [outlines-dev/outlines#1320] [Q&A] [answered]
When I use outlines on Llama3-8b, I find it reports an error:

_ValueError: The following `model_kwargs` are not used by the model: ['tokenizer'] (note: typos in the generate arguments will also show up in this list)_

After searching for a solution, we found that the only way to solve the above problem is to lower the transformer version to 4.21.3. Therefore, outlines can only be adapted to lower versions of transformers. However, Llama series LLMs generally run in higher transformer versions than this.

Python 3.9.19 (main, May  6 2024, 19:43:03) 
outlines == 0.1.7
transformers == 4.33.1

**Comment by ZhenbinChan** [comment, marked as answer]
I have solved this problem by reinstalling the outline with  `pip install "outlines[transformers]"`. I hope it will help some of u.

**Reply by just-cameron** [reply]
Thanks for adding your answer!
